umbraco/Umbraco-CMS
# V9: Displaying descendants in use as part of item tracking

## Details
- With this PR we are able to show **which** Content or Media descendants are referenced in other places of Umbraco.
- This improves the displayed information - before if click on a folder/parent item, it will only indicate that "_One or more of this item's descendants is being used in a content item_." in text.
- You can select an item from the list of "_Descendants in use_" and navigate to its info tab to observe the specific references.

## Test
- Have the SK installed
  - Pick a parent that has descendants and navigate to its info tab;
  - Observe if any nodes are listed as  "_Descendants in use_";
  - You can also try to delete/unpublish a parent and you will get the same table of "_Descendants in use_" + a warning message;
  - Make sure that the rest of the Item tracking functionality still works (listed in PR:#11919).
